chain/client/client.go

```diff
@@ -230,6 +230,7 @@ func ServeSwaggerUI() http.Handler {
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/d3n/blocks/latest", LatestBlocksRequestHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/d3n/txs/latest", LatestTxsRequestHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/staking/validators", GetValidators(cliCtx)).Methods("GET")
```

Comment:
Won't this collide with the existing `/staking/validators`? Let's call this `/bandchain/evm-validators`